terragrunt/aws/alarms/alarms.tf

```diff
@@ -4,17 +4,77 @@ resource "aws_cloudwatch_metric_alarm" "route53_health_check_api" {
   alarm_name          = "ScanFilesAPIHealthCheck"
   alarm_description   = "Check that Scan Files API is healthy"
   comparison_operator = "LessThanThreshold"
-  metric_name         = "HealthCheckStatus"
-  namespace           = "AWS/Route53"
-  period              = "60"
-  evaluation_periods  = "2"
-  statistic           = "Average"
-  threshold           = "1"
-  treat_missing_data  = "breaching"
+
+  metric_name        = "HealthCheckStatus"
+  namespace          = "AWS/Route53"
+  period             = "60"
+  evaluation_periods = "2"
+  statistic          = "Average"
+  threshold          = "1"
+  treat_missing_data = "breaching"
 
   alarm_actions = [aws_sns_topic.cloudwatch_warning_us_east.arn]
+  ok_actions    = [aws_sns_topic.cloudwatch_warning_us_east.arn]
 
   dimensions = {
     HealthCheckId = var.route53_health_check_api_id
   }
 }
+
+resource "aws_cloudwatch_log_metric_filter" "scan_files_api_error" {
+  name           = "ErrorLoggedAPI"
```

Comment:
opportunity to move `ErrorLoggedAPI` and `ErrorLoggedS3ScanObject` into locals to avoid duplication.

Reply:
Nice, I like that idea - added!